src/Command/ColdStorageFlagCommand.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace App\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use App\Entity\DatasetSubmission;
+use App\Entity\Dataset;
+use App\Entity\Person;
+
+use App\Util\RabbitPublisher;
+
+/**
+ * This command marks a dataset as cold-stored.
+ *
+ * This command saves the original hash and size, replaces the datafile with a
+ * supplied manifest file, then re-triggers filer/hasher in an expected way for
+ * a replaced datafile.
+ */
+class ColdStorageFlagCommand extends Command
+{
+    /**
+     * The Command name.
+     *
+     * @var string $defaultName
+     */
+    protected static $defaultName = 'pelagos:dataset-flag-coldstorage';
+
+    /**
+     * A Doctrine ORM EntityManager instance.
+     *
+     * @var EntityManagerInterface $entityManager
+     */
+    protected $entityManager;
+
+    /**
+     * Utility Rabbitmq producer instance.
+     *
+     * @var RabbitPublisher $publisher
+     */
+    protected $publisher;
+
+    /**
+     * Class constructor for dependency injection.
+     *
+     * @param EntityManagerInterface $entityManager A Doctrine EntityManager.
+     * @param RabbitPublisher        $publisher     A Rabbitmq producer instance.
+     */
+    public function __construct(EntityManagerInterface $entityManager, RabbitPublisher $publisher)
+    {
+        $this->entityManager = $entityManager;
+        $this->publisher = $publisher;
+        parent::__construct();
+    }
+
+    /**
+     * Symfony command config section.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Marks dataset in input file is cold-stored and updates datafile')
+            ->addArgument('infofile', InputArgument::REQUIRED, 'Filename of the coldinfo-formatted textfile required.');
```

Comment:
Where does this file come from? @fightingtexasaggie

Reply:
it is generated by a python script that uses the hashdeep file.

Reply:
verified (with an older db dump) this mailer works. 
(modify config/packages/dev/swiftmailer.yaml to test)